src/cursor/cur_index.c

```diff
@@ -452,8 +452,9 @@ __wt_curindex_open(WT_SESSION_IMPL *session, const char *uri, WT_CURSOR *owner,
       __wt_cursor_notsup,                             /* update */
       __wt_cursor_notsup,                             /* remove */
       __wt_cursor_notsup,                             /* reserve */
-      __wt_cursor_reconfigure_notsup,                 /* reconfigure */
+      __wt_cursor_config_notsup,                      /* reconfigure */
       __wt_cursor_notsup,                             /* largest_key */
+      __wt_cursor_config_notsup,                      /* bound */
```

Comment:
I forgot if we were going to support bounding of WT indices.  It wouldn't be hard to do.

Reply:
I'm not entirely sure if we are planning to support WT indices. From my knowledge we are not going ahead with supporting indices but I believe @luke-pearson can answer with more information as well!

Reply:
I'm going to defer to @agorrod here, however I am inclined to not support it in this project as indices weren't in the original scope and would require additional testing.

Reply:
I'd prefer to plan to support them, but am OK to cut scope if that becomes a significant amount of work.

Reply:
Alright we'll consider WT indices as in scope for the project.